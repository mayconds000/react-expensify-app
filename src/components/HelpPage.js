import React, { Component } from 'react';

const HelpPage = () => (
  <div>
    Help Expanse component
  </div>
);

export default HelpPage;