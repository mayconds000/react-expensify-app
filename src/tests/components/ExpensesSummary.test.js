import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from "../../components/ExpensesSummary";  

test('shoudl correctly render ExpenseSummary with 1 expense', () => {
  const wrapper = shallow(<ExpenseSummary expenseCount={1} expensesTotal={213} />);
  expect(wrapper).toMatchSnapshot();
});

test('shoudl correctly render ExpenseSummary with multiple expenses', () => {
  const wrapper = shallow(<ExpenseSummary expenseCount={35} expensesTotal={21312345687} />);
  expect(wrapper).toMatchSnapshot();
});