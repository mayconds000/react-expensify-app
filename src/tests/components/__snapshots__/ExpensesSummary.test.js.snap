// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shoudl correctly render ExpenseSummary with 1 expense 1`] = `
<div>
  <h1>
    viewing 
    1
     
    expense
     totalling. 
    $2.13
  </h1>
  ;
</div>
`;

exports[`shoudl correctly render ExpenseSummary with multiple expenses 1`] = `
<div>
  <h1>
    viewing 
    35
     
    expenses
     totalling. 
    $213,123,456.87
  </h1>
  ;
</div>
`;
